diff --git a/app/scripts/metamask-controller.js b/app/scripts/metamask-controller.js
index 3f1511333e6be632dca96c651ac94cb4cedd6f26..b6ae733ca2809ee5f44ec6389c599235bdd9f2d5 100644
--- a/app/scripts/metamask-controller.js
+++ b/app/scripts/metamask-controller.js
@@ -336,6 +336,10 @@ export default class MetamaskController extends EventEmitter {
     ) {
       this.submitPassword(password)
     }
+
+    if (chrome.braveWallet.hasOwnProperty('ready')) { // eslint-disable-line no-undef
+      chrome.braveWallet.ready() // eslint-disable-line no-undef
+    }
   }
 
   /**
@@ -571,6 +575,8 @@ export default class MetamaskController extends EventEmitter {
       addPermittedAccount: nodeify(permissionsController.addPermittedAccount, permissionsController),
       removePermittedAccount: nodeify(permissionsController.removePermittedAccount, permissionsController),
       requestAccountsPermissionWithId: nodeify(permissionsController.requestAccountsPermissionWithId, permissionsController),
+
+      setBatTokenAdded: nodeify(this.preferencesController.setBatTokenAdded, this.preferencesController),
     }
   }
 
