diff --git a/app/scripts/controllers/preferences.js b/app/scripts/controllers/preferences.js
index 47fa73746bf4cb1c0cee900564f9d726effea6d3..77b4b2d79c9df8756ec8daac97606928a2a1f5c4 100644
--- a/app/scripts/controllers/preferences.js
+++ b/app/scripts/controllers/preferences.js
@@ -62,6 +62,8 @@ export default class PreferencesController {
 
       // ENS decentralized website resolution
       ipfsGateway: 'dweb.link',
+
+      batTokenNetworks: {},
     }, opts.initState)
 
     this.diagnostics = opts.diagnostics
@@ -639,6 +641,13 @@ export default class PreferencesController {
     return Promise.resolve(domain)
   }
 
+  setBatTokenAdded (network) {
+    const batNetworks = this.store.getState().batTokenNetworks
+    batNetworks[network] = true
+    this.store.updateState({ batTokenNetworks: batNetworks })
+    return Promise.resolve(true)
+  }
+
   //
   // PRIVATE METHODS
   //
