diff --git a/app/scripts/background.js b/app/scripts/background.js
index adbd1e40ed73337f269e48fd9291ab8d9eac9bff..e0d4095c31ff0b3f512b13fac2ca96fd5afa415f 100644
--- a/app/scripts/background.js
+++ b/app/scripts/background.js
@@ -457,10 +457,3 @@ async function openPopup () {
     },
   )
 }
-
-// On first install, open a new tab with MetaMask
-extension.runtime.onInstalled.addListener(({ reason }) => {
-  if (reason === 'install' && !(process.env.METAMASK_DEBUG || process.env.IN_TEST)) {
-    platform.openExtensionInBrowser()
-  }
-})
