diff --git a/development/build/index.js b/development/build/index.js
index a665410b265aa104df4b8b7949059d98b4964c6c..484c21c9cdb1eeffed36fc4bc77982000ae78393 100755
--- a/development/build/index.js
+++ b/development/build/index.js
@@ -11,6 +11,9 @@ const createScriptTasks = require('./scripts')
 const createStyleTasks = require('./styles')
 const createStaticAssetTasks = require('./static')
 const createEtcTasks = require('./etc')
+const createBraveHTMLTasks = require('../../brave/build/html')
+const createBraveLoadStylesTasks = require('../../brave/build/styles')
+const createBraveManifestTasks = require('../../brave/build/manifest')
 
 const browserPlatforms = [
   'firefox',
@@ -30,15 +33,22 @@ function defineAllTasks () {
   const scriptTasks = createScriptTasks({ livereload, browserPlatforms })
   const { clean, reload, zip } = createEtcTasks({ livereload, browserPlatforms })
 
+  const braveHTMLTasks = createBraveHTMLTasks()
+  const braveLoadStylesTasks = createBraveLoadStylesTasks()
+  const braveManifestTasks = createBraveManifestTasks()
+
   // build for development (livereload)
   createTask('dev',
     composeSeries(
       clean,
       styleTasks.dev,
+      braveLoadStylesTasks,
+      staticTasks.dev,
+      manifestTasks.dev,
+      braveHTMLTasks,
+      braveManifestTasks,
       composeParallel(
         scriptTasks.dev,
-        staticTasks.dev,
-        manifestTasks.dev,
         reload,
       ),
     ),
