diff --git a/development/build/index.js b/development/build/index.js
index a665410b265aa104df4b8b7949059d98b4964c6c..e388fe9ea354666a9cb89fe9dcfd2f48ec37792c 100755
--- a/development/build/index.js
+++ b/development/build/index.js
@@ -11,6 +11,11 @@ const createScriptTasks = require('./scripts')
 const createStyleTasks = require('./styles')
 const createStaticAssetTasks = require('./static')
 const createEtcTasks = require('./etc')
+const createBraveHTMLTasks = require('../../brave/build/html')
+const createBraveLoadStylesTasks = require('../../brave/build/styles')
+const createBraveManifestTasks = require('../../brave/build/manifest')
+const createBraveLocalesTasks = require('../../brave/build/locales')
+const { fontsTask, imagesTask, fontsTaskProd, imagesTaskProd } = require('../../brave/build/assets')
 
 const browserPlatforms = [
   'firefox',
@@ -30,20 +35,52 @@ function defineAllTasks () {
   const scriptTasks = createScriptTasks({ livereload, browserPlatforms })
   const { clean, reload, zip } = createEtcTasks({ livereload, browserPlatforms })
 
+  const braveHTMLTasks = createBraveHTMLTasks()
+  const braveLoadStylesTasks = createBraveLoadStylesTasks()
+  const braveManifestTasks = createBraveManifestTasks()
+  const { stringTask, combineTask } = createBraveLocalesTasks()
+
   // build for development (livereload)
   createTask('dev',
     composeSeries(
       clean,
       styleTasks.dev,
+      braveLoadStylesTasks,
+      staticTasks.dev,
+      manifestTasks.dev,
+      braveHTMLTasks,
+      braveManifestTasks,
+      stringTask,
+      combineTask,
+      fontsTask,
+      imagesTask,
       composeParallel(
         scriptTasks.dev,
-        staticTasks.dev,
-        manifestTasks.dev,
         reload,
       ),
     ),
   )
 
+  createTask('prod:brave',
+    composeSeries(
+      clean,
+      styleTasks.prod,
+      braveLoadStylesTasks,
+      staticTasks.prod,
+      manifestTasks.prod,
+      braveHTMLTasks,
+      braveManifestTasks,
+      stringTask,
+      combineTask,
+      fontsTaskProd,
+      imagesTaskProd,
+      composeParallel(
+        scriptTasks.prod,
+      ),
+      zip,
+    ),
+  )
+
   // build for test development (livereload)
   createTask('testDev',
     composeSeries(
