diff --git a/development/build/styles.js b/development/build/styles.js
index 4667100c8a65d7de75cd35a8bdd5a185b5fc06c2..a81dc9f287c543d8e5de62630a5deadbdde018e7 100644
--- a/development/build/styles.js
+++ b/development/build/styles.js
@@ -9,12 +9,38 @@ const sourcemaps = require('gulp-sourcemaps')
 const rtlcss = require('gulp-rtlcss')
 const rename = require('gulp-rename')
 const pump = pify(require('pump'))
+const glob = require('fast-glob')
+const replace = require('gulp-replace')
 const { createTask } = require('./task')
 
 
 // scss compilation and autoprefixing tasks
 module.exports = createStyleTasks
 
+const getImportedPath = (path) => {
+  const mmPath = path.split('brave/')[1]
+  const braveStyleRelative = '../../../brave/'
+
+  return `@import '${braveStyleRelative}${mmPath}';`
+}
+
+async function braveStylesTask () {
+  const braveScss = 'brave/components/**/*.scss'
+  const appRootScss = 'ui/app/css/index.scss'
+  const braveImports = glob.sync(braveScss)
+    .map(getImportedPath)
+  await new Promise((resolve, _reject) => {
+    gulp.src(appRootScss)
+    .pipe(
+      replace(
+        /(?:\/\*BRAVE\*\/.*)?$/,
+        `/*BRAVE*/${braveImports.join('')}`,
+      ),
+    )
+    .pipe(gulp.dest((file) => file.base))
+    .on('end', resolve)    
+  })
+}
 
 function createStyleTasks ({ livereload }) {
 
@@ -47,6 +73,7 @@ function createStyleTasks ({ livereload }) {
 
   function createScssBuildTask ({ src, dest, devMode, pattern }) {
     return async function () {
+      await braveStylesTask()
       if (devMode) {
         watch(pattern, async (event) => {
           await buildScss(devMode)
