diff --git a/development/build/styles.js b/development/build/styles.js
index 4667100c8a65d7de75cd35a8bdd5a185b5fc06c2..9c971f6df39a199ac15efde14853e89fff92909e 100644
--- a/development/build/styles.js
+++ b/development/build/styles.js
@@ -28,7 +28,7 @@ function createStyleTasks ({ livereload }) {
     src: 'ui/app/css/index.scss',
     dest: 'ui/app/css/output',
     devMode: true,
-    pattern: 'ui/app/**/*.scss',
+    pattern: [ 'ui/app/**/*.scss', 'brave/components/**/*.scss' ],
   }))
 
   const lint = createTask('lint-scss', function () {
