diff --git a/ui/app/ducks/metamask/metamask.js b/ui/app/ducks/metamask/metamask.js
index e3f35bf38e960fb4ed60ba92ececdc597386ed5b..16ad18858a2de325a9629d43d1851dfe9af83f96 100644
--- a/ui/app/ducks/metamask/metamask.js
+++ b/ui/app/ducks/metamask/metamask.js
@@ -46,6 +46,7 @@ export default function reduceMetamask (state = {}, action) {
     participateInMetaMetrics: null,
     metaMetricsSendCount: 0,
     nextNonce: null,
+    batTokenNetworks: {},
   }, state)
 
   switch (action.type) {
@@ -362,6 +363,17 @@ export default function reduceMetamask (state = {}, action) {
       }
     }
 
+    case actionConstants.SET_BAT_TOKEN_ADDED: {
+      const network = action.value
+      return {
+        ...metamaskState,
+        batTokenNetworks: {
+          ...metamaskState.batTokenNetworks,
+          network: true
+        },
+      }
+    }
+
     default:
       return metamaskState
   }
