diff --git a/ui/app/pages/settings/networks-tab/networks-tab.constants.js b/ui/app/pages/settings/networks-tab/networks-tab.constants.js
index 453a616b209b964e8bc3b08ff711ba217d851d3d..79cf88d38b1cf12df254cc29729d338d955e7a23 100644
--- a/ui/app/pages/settings/networks-tab/networks-tab.constants.js
+++ b/ui/app/pages/settings/networks-tab/networks-tab.constants.js
@@ -1,4 +1,4 @@
-const defaultNetworksData = [
+const _defaultNetworksData = [
   {
     labelKey: 'mainnet',
     iconColor: '#29B6AF',
@@ -54,6 +54,13 @@ const defaultNetworksData = [
   },
 ]
 
+const defaultNetworksData = _defaultNetworksData.map((net) => {
+  return {
+    ...net,
+    rpcUrl: `${net.providerType}.infura.io/v3`,
+  }
+})
+
 export {
   defaultNetworksData,
 }
