diff --git a/ui/app/components/app/token-list/token-list.js b/ui/app/components/app/token-list/token-list.js
index 591dbe33af1245174c346bb4d9b1e81db32a4469..9846de3344637e463176d10cc0104aeb3074eb90 100644
--- a/ui/app/components/app/token-list/token-list.js
+++ b/ui/app/components/app/token-list/token-list.js
@@ -16,23 +16,7 @@ export default function TokenList ({ onTokenClick }) {
   // from the background so it has a new reference with each background update,
   // even if the tokens haven't changed
   const tokens = useSelector(getTokens, isEqual)
-  const { loading, error, tokensWithBalances } = useTokenTracker(tokens)
-
-  if (loading) {
-    return (
-      <div
-        style={{
-          display: 'flex',
-          height: '250px',
-          alignItems: 'center',
-          justifyContent: 'center',
-          padding: '30px',
-        }}
-      >
-        {t('loadingTokens')}
-      </div>
-    )
-  }
+  const { error, tokensWithBalances } = useTokenTracker(tokens)
 
   return (
     <div>
