diff --git a/ui/app/pages/first-time-flow/select-action/select-action.component.js b/ui/app/pages/first-time-flow/select-action/select-action.component.js
index 5379952f1aaf24ec7edcdf4c5511067ac4b996d7..8bf4a085bd409c73d85016c19aaeaa0b16732f4e 100644
--- a/ui/app/pages/first-time-flow/select-action/select-action.component.js
+++ b/ui/app/pages/first-time-flow/select-action/select-action.component.js
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types'
 import Button from '../../../components/ui/button'
 import MetaFoxLogo from '../../../components/ui/metafox-logo'
 import {
-  INITIALIZE_METAMETRICS_OPT_IN_ROUTE,
+  INITIALIZE_CREATE_PASSWORD_ROUTE,
 } from '../../../helpers/constants/routes'
 
 export default class SelectAction extends PureComponent {
@@ -28,12 +28,12 @@ export default class SelectAction extends PureComponent {
 
   handleCreate = () => {
     this.props.setFirstTimeFlowType('create')
-    this.props.history.push(INITIALIZE_METAMETRICS_OPT_IN_ROUTE)
+    this.props.history.push(INITIALIZE_CREATE_PASSWORD_ROUTE)
   }
 
   handleImport = () => {
     this.props.setFirstTimeFlowType('import')
-    this.props.history.push(INITIALIZE_METAMETRICS_OPT_IN_ROUTE)
+    this.props.history.push(INITIALIZE_CREATE_PASSWORD_ROUTE)
   }
 
   render () {
