diff --git a/ui/app/hooks/useTokenTracker.js b/ui/app/hooks/useTokenTracker.js
index 406eade3fb05985a4fbe9fc1632f9814cb18c14c..36e74b7dfe02f0c4b10041a97243f82365b1c5a0 100644
--- a/ui/app/hooks/useTokenTracker.js
+++ b/ui/app/hooks/useTokenTracker.js
@@ -40,7 +40,7 @@ export function useTokenTracker (tokens) {
       userAddress: address,
       provider: global.ethereumProvider,
       tokens: tokenList,
-      pollingInterval: 8000,
+      pollingInterval: (16 * 1000),
     })
 
     tokenTracker.current.on('update', updateBalances)
