diff --git a/ui/app/components/app/modals/modal.js b/ui/app/components/app/modals/modal.js
index 812090f10f461623e911ef2d96c45452f48824e8..d0ce205bbdfbb9b527831cac6b929ec15c2b4bd0 100644
--- a/ui/app/components/app/modals/modal.js
+++ b/ui/app/components/app/modals/modal.js
@@ -417,11 +417,15 @@ class Modal extends Component {
   }
 
   hide () {
-    this.modalRef.hide()
+    if (this.modalRef) {
+      this.modalRef.hide()
+    }
   }
 
   show () {
-    this.modalRef.show()
+    if (this.modalRef) {
+      this.modalRef.show()
+    }
   }
 
   UNSAFE_componentWillReceiveProps (nextProps, _) {
@@ -438,6 +442,10 @@ class Modal extends Component {
     const modalStyle = modal[isMobileView() ? 'mobileModalStyle' : 'laptopModalStyle']
     const contentStyle = modal.contentStyle || {}
 
+    if (this.props.modalState.name === 'METAMETRICS_OPT_IN_MODAL') {
+      return null
+    }
+
     return (
       <FadeModal
         keyboard={false}
