diff --git a/ui/app/components/ui/metafox-logo/metafox-logo.component.js b/ui/app/components/ui/metafox-logo/metafox-logo.component.js
index 4b1509c6ecbfe3b588a107d956bea2686a822c62..bf007228f3975c90a51ae145c11ca567512be891 100644
--- a/ui/app/components/ui/metafox-logo/metafox-logo.component.js
+++ b/ui/app/components/ui/metafox-logo/metafox-logo.component.js
@@ -12,7 +12,7 @@ export default class MetaFoxLogo extends PureComponent {
     onClick: undefined,
   }
 
-  render () {
+  _render () {
     const { onClick, unsetIconHeight } = this.props
     const iconProps = unsetIconHeight ? {} : { height: 42, width: 42 }
 
@@ -36,4 +36,8 @@ export default class MetaFoxLogo extends PureComponent {
       </div>
     )
   }
+
+  render () {
+    return null
+  }
 }
