diff --git a/ui/app/pages/home/home.container.js b/ui/app/pages/home/home.container.js
index 8a1c0eed77aac737942c1f81ec414544ac0f41e0..d0b7ff9d4b8f74641b23b4f8e018e7d92c49067e 100644
--- a/ui/app/pages/home/home.container.js
+++ b/ui/app/pages/home/home.container.js
@@ -4,7 +4,6 @@ import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
 import {
   unconfirmedTransactionsCountSelector,
-  getCurrentEthBalance,
   getFirstPermissionRequest,
   getTotalUnapprovedCount,
 } from '../../selectors'
@@ -29,14 +28,12 @@ const mapStateToProps = (state) => {
   const {
     suggestedTokens,
     seedPhraseBackedUp,
-    tokens,
     threeBoxSynced,
     showRestorePrompt,
     selectedAddress,
     connectedStatusPopoverHasBeenShown,
     defaultHomeActiveTabName,
   } = metamask
-  const accountBalance = getCurrentEthBalance(state)
   const { forgottenPassword, threeBoxLastUpdated } = appState
   const totalUnapprovedCount = getTotalUnapprovedCount(state)
 
@@ -53,7 +50,7 @@ const mapStateToProps = (state) => {
     forgottenPassword,
     suggestedTokens,
     unconfirmedTransactionsCount: unconfirmedTransactionsCountSelector(state),
-    shouldShowSeedPhraseReminder: !seedPhraseBackedUp && (parseInt(accountBalance, 16) > 0 || tokens.length > 0),
+    shouldShowSeedPhraseReminder: !seedPhraseBackedUp,
     isPopup,
     isNotification,
     threeBoxSynced,
