diff --git a/ui/app/pages/routes/routes.component.js b/ui/app/pages/routes/routes.component.js
index 1dc556a9828835e5a2581423a4966ad27d6c92c0..270946c4422ae1be695d5610dc72e24c8f579d61 100644
--- a/ui/app/pages/routes/routes.component.js
+++ b/ui/app/pages/routes/routes.component.js
@@ -31,6 +31,7 @@ import AppHeader from '../../components/app/app-header'
 import UnlockPage from '../unlock-page'
 import Alerts from '../../components/app/alerts'
 import Asset from '../asset'
+import BraveNavigation from '../../../../brave/components/navigation'
 
 import {
   ADD_TOKEN_ROUTE,
@@ -81,6 +82,8 @@ export default class Routes extends Component {
     hasPermissionsRequests: PropTypes.bool,
     autoLockTimeLimit: PropTypes.number,
     pageChanged: PropTypes.func.isRequired,
+    batTokenNetworks: PropTypes.object,
+    addBatToken: PropTypes.func.isRequired,
   }
 
   static contextTypes = {
@@ -88,6 +91,37 @@ export default class Routes extends Component {
     metricsEvent: PropTypes.func,
   }
 
+  componentDidMount () {
+    const { addBatToken, batTokenNetworks, network } = this.props
+    const container = document.querySelector('#app-content')
+
+    if (!container) {
+      return
+    }
+
+    const setTheme = (t) => {
+      container.className = `${t.toLowerCase()}`
+    }
+
+    if (chrome.hasOwnProperty('braveTheme')) {  // eslint-disable-line no-undef
+      chrome.braveTheme.getBraveThemeType((type) => setTheme(type))  // eslint-disable-line no-undef
+      chrome.braveTheme.onBraveThemeTypeChanged.addListener((type) => setTheme(type))  // eslint-disable-line no-undef
+    }
+
+    if (!(network in batTokenNetworks)) {
+      addBatToken(network)
+    }
+  }
+
+  componentDidUpdate (prevProps) {
+    const { addBatToken, batTokenNetworks, network } = this.props
+    if (prevProps.network !== this.props.network) {
+      if (!(network in batTokenNetworks)) {
+        addBatToken(network)
+      }      
+    }
+  }
+
   UNSAFE_componentWillMount () {
     const { currentCurrency, pageChanged, setCurrentCurrencyToUSD } = this.props
 
@@ -217,53 +251,56 @@ export default class Routes extends Component {
       !submittedPendingTransactions.find(({ id }) => id === sidebarTransaction.id)
 
     return (
-      <div
-        className={classnames('app', { 'mouse-user-styles': isMouseUser })}
-        dir={textDirection}
-        onClick={() => setMouseUserState(true)}
-        onKeyDown={(e) => {
-          if (e.keyCode === 9) {
-            setMouseUserState(false)
-          }
-        }}
-      >
-        <Modal />
-        <Alert
-          visible={this.props.alertOpen}
-          msg={alertMessage}
-        />
-        { !this.hideAppHeader() && (
-          <AppHeader
-            hideNetworkIndicator={this.onInitializationUnlockPage()}
-            disabled={this.onConfirmPage()}
+      <>
+        <BraveNavigation />
+        <div
+          className={classnames('app', { 'mouse-user-styles': isMouseUser })}
+          dir={textDirection}
+          onClick={() => setMouseUserState(true)}
+          onKeyDown={(e) => {
+            if (e.keyCode === 9) {
+              setMouseUserState(false)
+            }
+          }}
+        >
+          <Modal />
+          <Alert
+            visible={this.props.alertOpen}
+            msg={alertMessage}
+          />
+          { !this.hideAppHeader() && (
+            <AppHeader
+              hideNetworkIndicator={this.onInitializationUnlockPage()}
+              disabled={this.onConfirmPage()}
+            />
+          ) }
+          <Sidebar
+            sidebarOpen={sidebarIsOpen}
+            sidebarShouldClose={sidebarShouldClose}
+            hideSidebar={this.props.hideSidebar}
+            transitionName={sidebarTransitionName}
+            type={sidebarType}
+            sidebarProps={sidebar.props}
           />
-        ) }
-        <Sidebar
-          sidebarOpen={sidebarIsOpen}
-          sidebarShouldClose={sidebarShouldClose}
-          hideSidebar={this.props.hideSidebar}
-          transitionName={sidebarTransitionName}
-          type={sidebarType}
-          sidebarProps={sidebar.props}
-        />
-        <NetworkDropdown
-          provider={provider}
-          frequentRpcListDetail={frequentRpcListDetail}
-        />
-        <AccountMenu />
-        <div className="main-container-wrapper">
-          { isLoading && <Loading loadingMessage={loadMessage} /> }
-          { !isLoading && isLoadingNetwork && <LoadingNetwork /> }
-          { this.renderRoutes() }
+          <NetworkDropdown
+            provider={provider}
+            frequentRpcListDetail={frequentRpcListDetail}
+          />
+          <AccountMenu />
+          <div className="main-container-wrapper">
+            { isLoading && <Loading loadingMessage={loadMessage} /> }
+            { !isLoading && isLoadingNetwork && <LoadingNetwork /> }
+            { this.renderRoutes() }
+          </div>
+          {
+            isUnlocked
+              ? (
+                <Alerts />
+              )
+              : null
+          }
         </div>
-        {
-          isUnlocked
-            ? (
-              <Alerts />
-            )
-            : null
-        }
-      </div>
+      </>
     )
   }
 
