diff --git a/ui/app/pages/home/home.component.js b/ui/app/pages/home/home.component.js
index 880e438e0be1787a6bb1ec9c0bb753a8e650a424..29449704a687034d2645a31847b42a1e0947f77b 100644
--- a/ui/app/pages/home/home.component.js
+++ b/ui/app/pages/home/home.component.js
@@ -54,6 +54,8 @@ export default class Home extends PureComponent {
     connectedStatusPopoverHasBeenShown: PropTypes.bool,
     defaultHomeActiveTabName: PropTypes.string,
     onTabClick: PropTypes.func.isRequired,
+    addBatToken: PropTypes.func,
+    batTokenAdded: PropTypes.bool,
   }
 
   state = {
@@ -68,6 +70,8 @@ export default class Home extends PureComponent {
       suggestedTokens = {},
       totalUnapprovedCount,
       unconfirmedTransactionsCount,
+      addBatToken,
+      batTokenAdded,
     } = this.props
 
     this.setState({ mounted: true })
@@ -80,6 +84,10 @@ export default class Home extends PureComponent {
     } else if (Object.keys(suggestedTokens).length > 0) {
       history.push(CONFIRM_ADD_SUGGESTED_TOKEN_ROUTE)
     }
+
+    if (!batTokenAdded) {
+      addBatToken()
+    }
   }
 
   static getDerivedStateFromProps (
