diff --git a/ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js b/ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js
index 18f496ca74e8cd3fd1c2b18771d20d0ff21cc4e1..930283c37b17ed806202e1a07e07240c407e0277 100644
--- a/ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js
+++ b/ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js
@@ -261,5 +261,9 @@ function insert (list, value, target, removeOld) {
     })
   }
 
+  if (nextList.length > 24) {
+    nextList.pop()
+  }
+
   return nextList
 }
