diff --git a/ui/app/components/app/dropdowns/network-dropdown.js b/ui/app/components/app/dropdowns/network-dropdown.js
index e578a8b7561ea2b8dee2507c40fcd0ff0b15dad5..91ca5d186297337d5c851e537a573cf05b9f53de 100644
--- a/ui/app/components/app/dropdowns/network-dropdown.js
+++ b/ui/app/components/app/dropdowns/network-dropdown.js
@@ -67,7 +67,7 @@ class NetworkDropdown extends Component {
   }
 
   handleClick (newProviderType) {
-    const { provider: { type: providerType }, setProviderType } = this.props
+    const { provider: { type: providerType }, setProviderType, } = this.props
     const { metricsEvent } = this.context
 
     metricsEvent({
