diff --git a/ui/app/pages/routes/routes.container.js b/ui/app/pages/routes/routes.container.js
index d4a638250f829a7125c1afa245fb5b5f22e55b7c..8545b507fbbec57cb67cd44489b7c3932a90512b 100644
--- a/ui/app/pages/routes/routes.container.js
+++ b/ui/app/pages/routes/routes.container.js
@@ -45,6 +45,7 @@ function mapStateToProps (state) {
     providerId: getNetworkIdentifier(state),
     autoLockTimeLimit,
     hasPermissionsRequests: hasPermissionRequests(state),
+    batTokenNetworks: state.metamask.batTokenNetworks,
   }
 }
 
