diff --git a/ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js b/ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js
index 18f496ca74e8cd3fd1c2b18771d20d0ff21cc4e1..2b2bb3c70c8f689b2eed15fdfbb8a94aeb786442 100644
--- a/ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js
+++ b/ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js
@@ -9,7 +9,7 @@ import {
 import { exportAsFile } from '../../../../helpers/utils/util'
 import DraggableSeed from './draggable-seed.component'
 
-const EMPTY_SEEDS = Array(12).fill(null)
+const EMPTY_SEEDS = Array(24).fill(null)
 
 export default class ConfirmSeedPhrase extends PureComponent {
   static contextTypes = {
@@ -261,5 +261,9 @@ function insert (list, value, target, removeOld) {
     })
   }
 
+  if (nextList.length > 24) {
+    nextList.pop()
+  }
+
   return nextList
 }
