diff --git a/ui/app/components/app/signature-request-original/signature-request-original.component.js b/ui/app/components/app/signature-request-original/signature-request-original.component.js
index 8587f024ed66a4c0c85b88c732e72285e3077b47..c4c1eae362c3754a6ef08cb38ac782d44f5668e5 100644
--- a/ui/app/components/app/signature-request-original/signature-request-original.component.js
+++ b/ui/app/components/app/signature-request-original/signature-request-original.component.js
@@ -234,7 +234,7 @@ export default class SignatureRequestOriginal extends Component {
                   className="request-signature__help-link"
                   onClick={() => {
                     global.platform.openTab({
-                      url: 'https://metamask.zendesk.com/hc/en-us/articles/360015488751',
+                      url: 'https://support.brave.com/hc/en-us/articles/360035488071-How-do-I-manage-my-Crypto-Wallets-',
                     })
                   }}
                 >
